scripts/release/fetch-latest-yaml.sh

```diff
@@ -57,9 +57,13 @@ fetch_engine_operator_yaml()
     echo "Success: Downloaded ${C_URL} into ${TEMP_RESP_FILE}"
   fi
 
+  sed -e 's@node-disk-manager:ci@node-disk-manager:1.7.0@' -i ${TEMP_RESP_FILE}
+  sed -e 's@node-disk-operator:ci@node-disk-operator:1.7.0@' -i ${TEMP_RESP_FILE}
+  sed -e 's@node-disk-exporter:ci@node-disk-exporter:1.7.0@' -i ${TEMP_RESP_FILE}
   sed -e 's@\:ci@'"\:$TAG"'@' -i ${TEMP_RESP_FILE}
```

Comment:
Do we also need to change the linux-utils image to 3.0.0?

Reply:
Also, when is this script run?

Reply:
This script is not yet used. Currently used to find the differences across multiple repos and have identified some and fixed in this PR.